mbuild/formats/lammpsdata.py

```diff
@@ -24,6 +24,8 @@ def write_lammpsdata(structure, filename):
         ParmEd structure object
     filename : str
         Path of the output file
+    atom_style: str
+        Style of atoms to use in LAMMPS simulation
```

Comment:
This docstring should be thorough, like the one in `compound.py`. Directly copying the text should be fine.